assets/src/edit-story/components/colorPicker/pointer.js

```diff
@@ -60,10 +66,9 @@ const Pointer = styled.div.attrs(({ currentColor, withAlpha }) => {
   }
 `;
 
-Pointer.defaultProps = {
-  currentRGB: {},
-  currentRGBA: {},
-  currentColor: {},
-};
+function Pointer(props) {
+  const { isRTL } = useConfig();
+  return <PointerElement {...props} isRTL={isRTL} />;
```

Comment:
```suggestion
  return <PointerElement {...props} $isRTL={isRTL} />;
```

So we don't pollute the DOM with this prop. Of course need to update the styled component above as well.

Reply:
`isRTL` is never a valid HTML prop for any element, so it's not relevant. This prop will never pollute the DOM output as React will filter it out as an invalid DOM property.

Transient props are only important for attributes, that *are* valid HTML attributes, but just not used as such. E.g. `title`, `offset`, `background`, `x` etc.

Reply:
One could argue that we should use transient naming for all props used only for S-C that aren't supposed to go into the DOM, as a rule, to avoid forgetting it when it's important, but I think that's overkill. I think we should just remember to do this for properties, where this is a problem - and try to remind each other if we forget.